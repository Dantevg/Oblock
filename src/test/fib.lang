{
	fib n => n < 2 and 1 or fib(n-1) + fib(n-2)
	print("fib 10 = " + fib 10)
}
