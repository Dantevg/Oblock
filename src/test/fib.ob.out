fib 10 = 89
